File: src/BusinessTemplates/CSModernBusinessApp/OpenSilverBusinessApplication.Client/Models/User.partial.cs
using System.ComponentModel;

namespace $ext_safeprojectname$.Models
{
    /// <summary>
    /// Extensions to the <see cref="User"/> class.
    /// </summary>
    public partial class User
    {
        #region Make DisplayName Bindable

        protected override void OnPropertyChanged(string propertyName)
        {
            base.OnPropertyChanged(propertyName);

            if (propertyName == "Name" || propertyName == "FriendlyName")
            {
                this.RaisePropertyChanged("DisplayName");
            }
        }

        #endregion Make DisplayName Bindable
    }
}

File: src/BusinessTemplates/VBModernBusinessApp/OpenSilverBusinessApplication.Client/Models/User.partial.vb
Imports System.ComponentModel

Namespace Models

    Partial Public Class User

        Protected Overrides Sub OnPropertyChanged(propertyName As String)

            MyBase.OnPropertyChanged(propertyName)

            If propertyName = "Name" Or propertyName = "FriendlyName" Then
                Me.RaisePropertyChanged("DisplayName")
            End If

        End Sub

    End Class

End Namespace